servers/publikator/graphql/resolvers/_mutations/publish.js

```diff
@@ -354,7 +354,7 @@ module.exports = async (
       doc.content.meta.dossier
     )
 
-    resolved.dossier = dossiers.pop()
+    Object.assign(resolved, { meta: { dossier: dossiers.pop() } })
```

Comment:
`Object.assign` is not a deep merge. As far as I understand if something would belong to a format and dossier you'd loose the dossier down below